reactioncommerce/example-storefront
# feat: add tracker pages, events and identify

Segment and google analytics support

Resolves #676 
Impact: **minor**
Type: **feature**

## Testing
1. Add a Segment key and a google track-id to the .env file
2. Start the project
3. See in the segment dashboard if the events are reported
